server/config/v2_deprecation.go

```diff
@@ -17,7 +17,7 @@ package config
 type V2DeprecationEnum string
 
 const (
-	// Default in v3.5.  Issues a warning if v2store have meaningful content.
+	// No longer supported in v3.6
 	V2_DEPR_0_NOT_YET = V2DeprecationEnum("not-yet")
```

Comment:
If it is not supported in 3.6, can we just remove it and remove the switch case at line 40?

Reply:
I'm also usually for removing the dead code, however in this case I think it's worth to leave for historical context. Of course when V2 is fully removed this whole file should be deleted, however I think it's worth to keep this so that someone reading this file understands all previous enum stages.

Reply:
Sure, it isn't a big deal. Feel free to keep it unchanged. 

It's OK because this file will eventually be removed.